py/desisurvey/utils.py

```diff
@@ -120,4 +131,51 @@ def angsep(ra1, dec1, ra2, dec2):
     cosDelta = np.sin(DEC1)*np.sin(DEC2) + np.cos(DEC1)*np.cos(DEC2)*np.cos(deltaRA)
     return np.degrees(np.arccos(cosDelta))
 
+def equ2gal_J2000(ra_deg, dec_deg):
+
+    ra = np.radians(ra_deg)
+    dec = np.radians(dec_deg)
+
+    x = np.empty(3, dtype='f8')
+    x[0] = np.cos(ra) * np.cos(dec)
+    x[1] = np.cos(ra) * np.sin(dec)
+    x[2] = np.sin(ra)
+
+    M = np.array([ [-0.054876, -0.873437, -0.483835],
+                   [ 0.494109, -0.444830,  0.746982],
+                   [-0.867666, -0.198076,  0.455984] ])
+
+    y = M.dot(x)
+    b = np.arcsin(y[2])
+    l = np.arctan2(y[1], y[0])
+
+    l_deg = np.degrees(l)
+    b_deg = np.degrees(b)
 
+    return l_deg, b_deg
+
+def sort2arr(a, b):
+    """Sorts array a according to the values of array b
+    """
+
+    if len(a) != len(b):
+        print("error: a and b are not of the same length.")
+        exit()
+
+    c = np.vstack((a,b)).T
+    d = c[np.argsort(c[:, 1])]
+    
+    return d[:,0]
```

Comment:
numpy recipe: this function can be implemented in a single line as `a[np.argsort(b)]`